source/shared/net/desktop-wallet/change-restake-settings.rst

```diff
@@ -9,9 +9,13 @@ By default, your baker rewards are restaked when you create the baker account. H
 Change restake settings (Single-signature account)
 ==================================================
 
-#. Go to **Accounts**, select the account whose baker keys you want to update, change the view to the detailed view, and select **Update baker restake earnings**.
+#. Go to **Accounts** and select the account whose baker stake you want to change.
```

Comment:
```suggestion
#. Go to **Accounts** and select the account whose baker restake preferences you want to change.
```
I guess? It is a little akward, because it is now together with the update stake and not its own transaction anymore.